doc/_static/mpl.css

```diff
@@ -23,6 +23,14 @@ a {
     text-decoration: none;
 }
 
+a.rcparams {
```

Comment:
Is is really worth it using a different styling (or even a different class)?
I think the normal orange would probably actually look better.

Reply:
If we want to use styling, a class is the cleanest way.

To me, the original orange looked a bit too distracting. Can add a screenshot later.

Reply:
Just for comparison. I still prefer the darker links for the rcParams. But I'm not dogmatic about this if the majority has another opinion.

## Same color as regular links

![grafik](https://user-images.githubusercontent.com/2836374/41562139-0c07c744-734c-11e8-89fb-c3d66812fb29.png)

## Darker links

![grafik](https://user-images.githubusercontent.com/2836374/41562186-2c3ab878-734c-11e8-9898-be6b28e49953.png)

Reply:
Let's see what others think, then.

Reply:
:+1: opinions welcome

Reply:
I'm not sure why it should be distinct.  Every other link in the docs, including normal url links, are the same orange.